libs/wire-api/src/Wire/API/Routes/Public/Brig.hs

```diff
@@ -16,16 +15,19 @@
 --
 -- You should have received a copy of the GNU Affero General Public License along
 -- with this program. If not, see <https://www.gnu.org/licenses/>.
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
I don't like this so if you have any suggestions where to put that instance for `AsUnion DeleteSelfResponses (Maybe Timeout)`, I am all ears.